src/Core/ObjectRegistry.php

```diff
@@ -103,6 +103,7 @@ public function load(string $objectName, array $config = [])
             }
         }
 
+        // phpcs:ignore SlevomatCodingStandard.Namespaces.FullyQualifiedClassNameInAnnotation.NonFullyQualifiedClassName
```

Comment:
I wish slevomat would stop trying to validate psalm annotations.

Reply:
We cannot write `\TObject` here?

Reply:
It might do a blind string comparison, but I would prefer it not look like a qualified type since it's meant to be a `T` generics type.